js-core/homeworks/homework-6/src/main.js

```diff
@@ -0,0 +1,524 @@
+		name: 'Jerri Richardson',
+		id: 13,
+	  },
+	  {
+		name: 'Burnett Sharp',
+		id: 14,
+	  },
+	  {
+		name: 'Katy Madden',
+		id: 15,
+	  },
+	  {
+		name: 'Kristine Payne',
+		id: 16,
+	  },
+	  {
+		name: 'Ashlee Wilson',
+		id: 17,
+	  },
+	  {
+		name: 'Jan Pugh',
+		id: 18,
+	  },
+	  {
+		name: 'Michael King',
+		id: 19,
+	  },
+	  {
+		name: 'Patty Rivas',
+		id: 20,
+	  },
+	  {
+		name: 'Bridges Oneil',
+		id: 21,
+	  },
+	  {
+		name: 'Walters Vazquez',
+		id: 22,
+	  },
+	  {
+		name: 'English Andrews',
+		id: 23,
+	  },
+	  {
+		name: 'Woodard Cardenas',
+		id: 24,
+	  },
+	  {
+		name: 'Mercado Chan',
+		id: 25,
+	  },
+	  {
+		name: 'Guerra Lawson',
+		id: 26,
+	  },
+	  {
+		name: 'Riley Gray',
+		id: 27,
+	  },
+	  {
+		name: 'Romero Sampson',
+		id: 28,
+	  },
+	  {
+		name: 'Reva Carroll',
+		id: 29,
+	  },
+	  {
+		name: 'Carey Mckenzie',
+		id: 30,
+	  },
+	  {
+		name: 'Miles Raymond',
+		id: 31,
+	  },
+	  {
+		name: 'Ray Underwood',
+		id: 32,
+	  },
+	  {
+		name: 'Maude Norris',
+		id: 33,
+	  },
+	  {
+		name: 'Brigitte Kirkland',
+		id: 34,
+	  },
+	  {
+		name: 'Graham Russell',
+		id: 35,
+	  },
+	  {
+		name: 'Sonia Holcomb',
+		id: 36,
+	  },
+	  {
+		name: 'Hopper Garrison',
+		id: 37,
+	  },
+	  {
+		name: 'Butler Harrison',
+		id: 38,
+	  },
+	  {
+		name: 'Grant Wheeler',
+		id: 39,
+	  },
+	  {
+		name: 'Stout Barker',
+		id: 40,
+	  },
+	  {
+		name: 'Merritt Kelley',
+		id: 41,
+	  },
+	  {
+		name: 'Kari Holman',
+		id: 42,
+	  },
+	  {
+		name: 'Tamra Howell',
+		id: 43,
+	  },
+	  {
+		name: 'Good Harvey',
+		id: 44,
+	  },
+	  {
+		name: 'Hendrix Webb',
+		id: 45,
+	  },
+	  {
+		name: 'Hammond Harris',
+		id: 46,
+	  },
+	  {
+		name: 'Ester Hamilton',
+		id: 47,
+	  },
+	  {
+		name: 'Lopez Hopper',
+		id: 48,
+	  },
+	  {
+		name: 'Newton Cantrell',
+		id: 49,
+	  },
+	],
+	range: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+	tags: ['nulla', 'Lorem', 'labore', 'dolore', 'enim'],
+	longitude: '0.298835',
+	latitude: '20.822214',
+	registered: 'Saturday, July 2, 2016 10:52 PM',
+	about: 'Ipsum adipisicing nulla adipisicing culpa qui et esse labore ut nostrud excepteur amet elit voluptate. Ut ex laborum enim sit. Proident eu elit tempor occaecat dolor.',
+	address: '597 Agate Court, Interlochen, Utah, 6828',
+	phone: '+1 (987) 453-2740',
+	email: 'lenora.barr@uberlux.me',
+	company: 'UBERLUX',
+	name: {
+	  last: 'Barr',
+	  first: 'Lenora',
+	},
+	eyeColor: 'brown',
+	age: 37,
+	picture: 'http://placehold.it/32x32',
+	balance: '$3,110.45',
+	isActive: true,
+	guid: '74d9074f-004a-447f-8721-12d884011852',
+	index: 0,
+	_id: '584bac50967253d4322db586',
+  };
+  
+  /*
+   *
+   * TASK 1
+   * Посчитайте количество букв а, в передаваемом параметре
+   *
+   * */
+  
+  let randomString = 'aaa b a w c ';
+  
+  let user = {
+	name: 'Albina'
+  };
+  
+  let javaScript = {
+	html: 'JavaScript'
+  };
+
+//console.log(countLetterA(user.name + javaScript.html));
+  
+  function countLetterA(str) {
+	  let result=``;
```

Comment:
oh, why do you make so many checks about the **str** argument?

Let's take it simple, that function always applies a string.